src/pages/Login/Login.js

```diff
@@ -1,5 +1,118 @@
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import LoginSignUpButton from '../../components/LoginSignUpButton/LoginSignUpButton';
+import HeaderMain from '../../components/HeaderMain/HeaderMain';
+import HeaderTop from '../../components/HeaderTop/HeaderTop';
+import InputBox from '../../components/InputBox/InputBox';
+import { CiSearch } from 'react-icons/ci';
 import './Login.scss';
+
 const Login = () => {
-  return <div className="login">로그인</div>;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    // fetch('https://10.58.52.172:3000/users/signup', {
+    //   method: 'POST',
+    //   headers: {
+    //     'Content-Type': 'application/json;charset=utf-8',
+    //   },
+    //   body: JSON.stringify({
+    //     email: userDataValue.email,
+    //     password: userDataValue.password,
+    //   }),
+    // })
+    //   .then(res => {
+    //     if (res.ok) {
+    //       return res.json();
+    //     }
+    //   })
+    //   .then(data => {
+    //     if (data.message === 'sucess login') {
+    //       localStorage.setItem('TOKEN', data.token);
+    //       alert('로그인 성공');
+    //       navigate('/product-list');
+    //     } else {
+    //       alert('로그인 실패');
+    //     }
+    //   });
+    navigate('/product-list');
+  };
+  const handleInput = e => {
+    const { type, value } = e.target;
+    console.log(e.target.value);
+    setUserDataValue({ ...userDataValue, [type]: value });
+    console.log(userDataValue);
+  };
+
+  const [userDataValue, setUserDataValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  let isVaild = () => {
+    return userDataValue.email.includes('@') &&
+      userDataValue.password.length >= 5
+      ? false
+      : true;
+  };
```

Comment:
- 함수가 아닌 변수 형태로 사용하시면 됩니다.
- 그리고, 결과가 반대로 나올 것 같네요!
- 또한, 조건이 boolean이기 때문에 조건 자체를 값으로 사용하시면 됩니다.
- 재할당하는 값이 아니기 때문에 const로 선언하시면 됩니다.

```suggestion
  const isValid = userDataValue.email.includes('@') &&
      userDataValue.password.length >= 5
```